docs/README.md

```diff
@@ -326,6 +327,32 @@ There are two methods for it:
 
 <br/>
 
+# 6. Cookies
+
+Cookies are the most efficient method of remembering and tracking preferences, purchases, commissions, and other information required for better visitor experience or site statistics.
+
+<hr />
+Cookies are a plain text data record of 5 variable-length fields −
+
+## Expires − 
+          The date the cookie will expire. If this is blank, the cookie will expire when the visitor quits the browser.
+
+## Domain − 
+          The domain name of your site.
+
+## Path − 
+          The path to the directory or web page that set the cookie. This may be blank if you want to retrieve the cookie from any directory or page.
+
+## Secure − 
+          If this field contains the word "secure", then the cookie may only be retrieved with a secure server. If this field is blank, no such restriction exists.
+
+## Name=Value − 
+          Cookies are set and retrieved in the form of key-value pairs
+          
```

Comment:
Also, it would be great if you could add a simple js program for creating, reading, updating, and deleting a cookie